src/main/java/com/capgemini/data/ReservationRepository.java

```diff
@@ -1,18 +1,36 @@
 package com.capgemini.data;
 
+import com.capgemini.domain.Guest;
 import com.capgemini.domain.Reservation;
-import java.util.ArrayList;
-import java.util.List;
+import com.capgemini.domain.Room;
+import com.capgemini.domain.RoomType;
+import org.springframework.stereotype.Component;
 
+import java.util.*;
```

Comment:
Kan je de specifieke class ff importen? anders compiled ie alles in java.util mee..